src/components/navBar/navBar.spec.js

```diff
@@ -364,13 +364,13 @@ describe('mdNavBar', function() {
 
     it('automatically adds label to nav items', function() {
       createTabs();
-      expect(getTab('tab1').parent().attr('aria-label')).toBe('tab1');
-      expect(getTab('tab2').parent().attr('aria-label')).toBe('tab2');
+      expect(getTab('tab1').attr('aria-label')).toBe('tab1');
+      expect(getTab('tab2').attr('aria-label')).toBe('tab2');
     });
 
     it('does not change aria-label on nav items', function() {
       createTabs();
-      expect(getTab('tab3').parent().attr('aria-label')).toBe('foo');
+      expect(getTab('tab3').attr('aria-label')).toBe('foo');
```

Comment:
To ensure that we don't break any apps, let's continue to check that the `aria-label` API still works the way it did before:
```js
    it('does not change aria-label on nav items', function() {
      createTabs();
      expect(getTab('tab3').parent().attr('aria-label')).toBe('foo');
    });
```
Then let's add a new check for the `nav-item-aria-label`
```js
    it('does not change nav-item-aria-label on nav item buttons', function() {
      createTabs();
      expect(getTab('tab4').attr('aria-label')).toBe('foo');
    });
```
This will also require adding another tab to `createTabs()` around line 47:
```js
        '  <md-nav-item md-nav-href="#4" name="tab4" nav-item-aria-label="foo">' +
        '    tab4' +
        '  </md-nav-item>' +
```